terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverEventListenerAdapter.java

```diff
@@ -83,4 +83,12 @@ public void beforeScript(String arg0, WebDriver arg1) {
     public void onException(Throwable arg0, WebDriver arg1) {
     }
 
+    @Override
+    public void afterNavigateRefresh(WebDriver arg0) {
+    }
+
+    @Override
+    public void beforeNavigateRefresh(WebDriver arg0) {
+    }
+

```

Comment:
Fixed with [Define same version dependencies for selenium-java](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/527/commits/b8af8c4b467fedaee65d7c11e199c657d7bf8449) and [Delete unnecessary abstract methods](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/527/commits/1a11c0bcd3cae2ee23a5f22384144da913c03bf1).